### DISCUSSION TITLE
there is a nasty bug in the last 10.1 builds

### DISCUSSION THREAD

**Discussion by antoninchadima** [ianw1974/zimbra-build-scripts#100] [General]
the address autocomplete is failing in the web ui,
if there is any contact without a email address

[Github](https://github.com/Zimbra/zm-mailbox/pull/1660/commits/aa71d7fcb29f365cf25b040396dff6d42757f545)

waiting for new builds...

**Comment by ianw1974** [comment]
I wouldn't classify the lack of autocomplete as a nasty bug.  Rather something that doesn't work as intended, so really purely cosmetic since Zimbra can still be used without autocomplete functionality working, and wouldn't classify it as serious either.

That said, builds will be available in due course.

**Reply by antoninchadima** [reply]
:+1: 
But, it is the first thing users complain about. :rofl:

**Reply by ianw1974** [reply]
Maybe, but it's still not critical or serious when compared for example to a vulnerability that gets a high CVE score.

Anyway the builds are up, feel free to download and hopefully the Zimbra fix made it from the develop branch if Zimbra commited it properly.
